README.md

```diff
@@ -96,12 +96,51 @@ export PATH="${PATH}:${HOME}/.local/bin/"
 
 UtkarshVerma was so kind to provide an installation script for debian based systems, ![check it out here](https://github.com/UtkarshVerma/installer-scripts).
```

Comment:
Repository archived, possibly remove hint as we provide installation instructions and in further versions make-process?

Reply:
If it's archived, go ahead and remove it